openmetadata-service/src/main/java/org/openmetadata/service/OpenMetadataApplication.java

```diff
@@ -340,6 +349,34 @@ private void validateMigrations(Jdbi jdbi, MigrationConfiguration conf) throws I
     }
   }
 
+  private void validateAndRunSystemDataMigrations(Jdbi jdbi) throws IOException {
+    // TODO: Add validation and migration if required
+    List<MigrationStep> loadedMigrationFiles = getServerMigrationFiles();
+    MigrationWorkflow workflow =
+        new MigrationWorkflow(jdbi, DatasourceConfig.getInstance().getDatabaseConnectionType(), loadedMigrationFiles);
+    workflow.runMigrationWorkflows();
+  }
+
+  private List<MigrationStep> getServerMigrationFiles() {
```

Comment:
@mohityadav766 lets move this to TablesInitializer not on server start

Reply:
resolved